app/utills.py

```diff
@@ -0,0 +1,26 @@
+import httpx
+import os
+from dotenv import load_dotenv
+
+
+load_dotenv()
```

Comment:
The `load_dotenv()` function is being called every time this module is imported. This could lead to unnecessary I/O operations if this module is imported multiple times. Consider moving the call to `load_dotenv()` to a startup script or the main entry point of your application.